sympy/plotting/plot.py

```diff
@@ -575,7 +575,7 @@ class LineOver1DRangeSeries(Line2DBaseSeries):
     def __init__(self, expr, var_start_end, **kwargs):
         super(LineOver1DRangeSeries, self).__init__()
         self.expr = sympify(expr)
-        self.label = str(self.expr)
+        self.label = kwargs.get('label', None) or str(self.expr)
```

Comment:
@sylee957 Should I make this change in a new PR? It involves adding `label` as an argument in the `plot` so that we could use them to denote when we use `legend` to name that line as in matplotlib. Currently, it by default considers `label` as `str(self.expr)` . I will look something like this.

![Screenshot from 2020-04-06 11-23-00](https://user-images.githubusercontent.com/55887635/78526930-2c2e1100-77f9-11ea-9465-640845947b22.png)

Reply:
I think that labeling a plot manually can be a useful feature. 
But you should add tests for ordinary plots to see how this works.

Reply:
Is the plotting module able to plot a set of functions in the same graph as shown above? Is the public API of plot capable of supporting such stuff?

Reply:
Yes, this is able to plot many functions in the same graph. Seeing this as a useful feature, I made a change in the` plotting` module. But, I think, I should open a new PR for this as it involves a change in public API of `plot`. Also, I think the current plot is unable to plot discrete graphs as we have scatter plot in matplotlib which is required for plotting discrete RV.